pczt/src/roles/creator/mod.rs

```diff
@@ -106,6 +106,8 @@ impl Creator {
             | zcash_primitives::transaction::TxVersion::Overwinter => None,
             zcash_primitives::transaction::TxVersion::Sapling => Some(SAPLING_TX_VERSION),
             zcash_primitives::transaction::TxVersion::Zip225 => Some(V5_TX_VERSION),
+            #[cfg(zcash_unstable = "zfuture")]
+            zcash_primitives::transaction::TxVersion::ZFuture => None,
```

Comment:
As an aside, I don't think it makes sense to name transaction versions after NUs or ZIPs. With the exception of `TxVersion::ZFuture`, they are just numbers.

Reply:
Agreed, we can fix `SAPLING_TX_VERSION` later.